uaclient/util.py

```diff
@@ -166,14 +167,13 @@ def readurl(url: str, data: 'Optional[bytes]' = None,
     return content, resp.headers
 
 
-def subp(args: 'Sequence[str]', rcs: 'Optional[List[int]]' = None,
-         capture: bool = False) -> 'Tuple[str, str]':
+def _subp(args, rcs=None, capture=False):
```

Comment:
Not having type annotations here means that the body of the function won't be type-checked (because `args`, `rcs` and `capture` will all be Any).